docs/ldml/tr35-numbers.md

```diff
@@ -1092,6 +1092,8 @@ Plural categories may also differ according to the visible decimals. For example
 
 There are also variants of the above: for example, short fractions may have the Digits behavior, but longer fractions may just look at the final digit of the fraction.
 
+Currently there are no locale keywords that affect plural rule selection; they are selected using the base locale ID, ignoring anu -u- extension keywords.
```

Comment:
anu ==> any

Reply:
@macchiati Thanks, I fixed there "anu" typo, please re-stamp (need to remove your merge block on this)